Header header

float64[] joint_position
float64[] joint_velocity

float64[] cartesian_position
float64[] cartesian_velocity